<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ClientInfo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>228</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>HY견고딕</family>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ONE 4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>228</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>ID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Name</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Date</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Equipment</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="findLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="enterPushButton">
             <property name="text">
              <string>ENTER</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTableView" name="clientTableView"/>
          <widget class="QGraphicsView" name="infoGraphicsView"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downLoadPushButton">
         <property name="text">
          <string>DownLoad</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Clinet</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableView" name="tableView"/>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGraphicsView" name="graphicsView"/>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&amp;ID</string>
                </property>
                <property name="buddy">
                 <cstring>idLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="idLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>&amp;Name</string>
                </property>
                <property name="buddy">
                 <cstring>nameLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nameLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>&amp;Gender</string>
                </property>
                <property name="buddy">
                 <cstring>genderLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="genderLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>&amp;Age</string>
                </property>
                <property name="buddy">
                 <cstring>ageLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ageLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="addPushButton">
                <property name="text">
                 <string>A&amp;DD</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="modifyPushButton">
                <property name="text">
                 <string>&amp;MODIFY</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
